dev/com.ibm.jbatch.container/bnd.bnd

```diff
@@ -16,7 +16,6 @@ Bundle-SymbolicName: com.ibm.jbatch.container
 Bundle-Description:  Java Batch RI core runtime implementation
 
 # java batch is part of EE7 and therefore requires java 1.7
```

Comment:
Need to remove this comment too

Reply:
The comment is actually accurate still.  I did not clean up all the comments in the code.  I automated this with scripting to make it possible to get this done efficiently instead of having to do it all by hand.  If you feel strongly about it, I can remove it.